add-sync-map/index.js

```diff
@@ -44,6 +44,14 @@ function buildFilter(filter) {
         if (action.fields[key] !== filter[key]) return false
       }
     }
+    if (action.type.endsWith('/changed')) {
+      for (let key in filter) {
+        if (
+          action.fields[key] &&
```

Comment:
```suggestion
          key in action.fields &&
```

Otherwise if will not work with `""` and `false`.